pkgs/applications/networking/sniproxy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, gettext, libev, pcre, pkgconfig, udns }:
+
+stdenv.mkDerivation rec {
+  name = "sniproxy-${version}";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "dlundquist";
+    repo = "sniproxy";
+    rev = "0.4.0";
+    sha256 = "1r6hv55k2z8l5q57l2q2x3nsspc2yjvi56l760yrz2c1hgh6r0a2";
+  };
+
+  buildInputs = [ autoconf automake gettext libev pcre pkgconfig udns ];
```

Comment:
Ah yes, just learned of autoreconfHook now. Will include this in my next rebase.